tests/test_api_test.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2020 by Christian <c.hagenest@pm.me>
+# Copyright (C) 2020 by Thomas Lindner <tom@dl6tom.de>
+#
+# SPDX-License-Identifier: 0BSD
+
+from fastapi import status
+
+
+def test_test_read_all_unauthorized(client, hood_id):
+    response = client.get('/api/hoods/%d/test/' % hood_id)
+    assert response.status_code == status.HTTP_401_UNAUTHORIZED
+
+
+def test_test_create_unauthorized(client, hood_id):
```

Comment:
And where is `client` and `hood_id` coming from here? Who calls these test functions?

Reply:
The functions are called by pytest. `client`/`hood_id`/... are fixtures defined globally in `conftest.py`. Fixtures do common setup/teardown, pytest calls them and passes the `return`ed or `yield`ed value to the test function.